Gemfile

```diff
@@ -32,6 +32,7 @@ gem 'jbuilder', '~> 2.5'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem 'pagy'
```

Comment:
@muhammadharoon63 has used Kaminari. Please coordinate with him.
We do not want two pagination gems in one project.